api/src/main/java/org/apache/commons/rdf/api/RDFSyntax.java

```diff
@@ -34,7 +152,7 @@
  *      1.1 Primer</a>
  * @see org.apache.commons.rdf.experimental.RDFParser
  */
-public enum RDFSyntax {
+  public enum OfficialRDFSyntax implements RDFSyntax {
```

Comment:
There is no particular reason that "enum" needs to be used given the interface design is done for flexibility, and given the difficulties with equals and hashCode. I would suggest reimplementing the "values()" method, along with equals and hashCode, and changing this to a "public final class".